logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/Message.kt

```diff
@@ -352,6 +352,22 @@ sealed interface Message {
                 is MessageContent.ConversationStartedUnverifiedWarning -> mutableMapOf(
                     typeKey to "conversationStartedUnverifiedWarning"
                 )
+
+                MessageContent.LegalHold.ForConversation.Disabled -> mutableMapOf(
+                    typeKey to "legalHoldDisabledForConversation"
+                )
+                is MessageContent.LegalHold.ForMembers.Disabled -> mutableMapOf(
+                    typeKey to "legalHoldDisabledForMembers",
+                    "members" to content.members.fold("") { acc, member ->
+                        "$acc, ${member.value.obfuscateId()}@${member.domain.obfuscateDomain()}"
+                    }
+                )
+                is MessageContent.LegalHold.ForMembers.Enabled -> mutableMapOf(
+                    typeKey to "legalHoldEnabledForMembers",
+                    "members" to content.members.fold("") { acc, member ->
+                        "$acc, ${member.value.obfuscateId()}@${member.domain.obfuscateDomain()}"
+                    }
+                )
```

Comment:
No, according AC and designs, for conversation we have only have a system message when the conversation is no longer under legal hold: https://www.figma.com/file/UgKehvwIbiIiQ2CCEdsVuf/Android?type=design&node-id=41079-16562&mode=dev

<img width="867" alt="image" src="https://github.com/wireapp/kalium/assets/30429749/789f75c0-a0da-458e-b382-457607f96f9d">

Reply:
Okay maybe we would need to sync with product because on web we display if legal hold is enabled in a conversation. "This conversation is under legal hold"